budget app backend/controllers/auth_controller.js

```diff
@@ -0,0 +1,29 @@
+module.exports = class{
```

Comment:
I dont understand why do you use classes and static fucntion (it's dont remark. I just don't understand)

Reply:
I used class beacuse it is easier to export and import. Instead of exporting several controller functions, I simply export class with static functions :)